### PR TITLE
device: add leobog hi75

### PR DESCRIPTION
**Pull request by Mooyeee**
https://github.com/carlossless/sinowealth-kb-tool/issues/73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,54 +68,55 @@ sinowealth-kb-tool write \
...
-| [Royal Kludge RK61](http://en.rkgaming.com/product/11/) | 3e0ebd0c440af5236d7ff8872343f85d | SH68F90? | BYK916 | ✅ | ✅ |
-| Royal Kludge RK68 BT Dual | cfc8661da8c9d7e351b36c0a763426aa | SH68F90? | BYK901 | ✅ | ✅ |
-| Royal Kludge RK68 ISO Return | ❓ | SH68F90? | BYK916 | ✅ | ❓ |
-| [Royal Kludge RK71](http://en.rkgaming.com/product/12/) | cfc8661da8c9d7e351b36c0a763426aa | SH68F90? | ❓ | ✅ | ✅ |
-| [Royal Kludge RK84](http://en.rkgaming.com/product/16/) | cfc8661da8c9d7e351b36c0a763426aa | SH68F90? | BYK916 | ✅ | ✅ |
-| Terport TR95 | 2d169670eae0d36eae8188562c1f66e8 | SH68F90A | BYK916 | ✅ | ✅ |
-| Weikav Sugar65 | 2d169670eae0d36eae8188562c1f66e8 | SH68F90 | SH68F90S | ✅ | ✅ |
-| Xinmeng K916 | cfc8661da8c9d7e351b36c0a763426aa | SH68F90 | ❓ | ✅ | ✅ |
-| Xinmeng XM-RF68 | 2d169670eae0d36eae8188562c1f66e8 | SH68F90 | SH68F90U | ✅ | ✅ |
+| Model                                                                                                                                                                    | ISP MD5                          | MCU      | MCU Label | Tested Read | Tested Write |
```
Comment (carlossless) [inline README.md:71]: Please don't reformat all of these tables. Just add your device info in the alphabetical order this list is sorted in.

**Review by carlossless — CHANGES_REQUESTED** [review]
Hey, thanks for the device report & PR!

Before I can merge this in though you should add the device info to src/part.rs and remove the table reformating in README.md

**Comment by Mooyeee** [comment]
Hey, my bad for the formatting, it was done by the software I use to edit md files (MarkText), now it should be ok. I also added the part as requested, let me know if it's alright or if there are other changes needed! :)
I'll try to experiment a little with smk too since this isn't my main board.

File: src/part.rs [hunk 2]
```diff
@@ -207,6 +214,7 @@ pub static PARTS: Map<&'static str, Part> = phf_map! {
     "genesis-thor-300-rgb" => PART_GENESIS_THOR_300_RGB,
     "genesis-thor-300" => PART_GENESIS_THOR_300,
     "glorious-model-o" => PART_GLORIOUS_MODEL_O,
+    "leobog-hi-75" => PART_LEOBOG_HI75,
```
Comment (carlossless) [inline src/part.rs:217]:
```suggestion
    "leobog-hi75" => PART_LEOBOG_HI75,
```

**Review by carlossless — APPROVED** [review]
LGTM

**Pull request merged**